### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by HydriDev** [AXDZ/NameMC-ViewBot#1]
HTTPSConnectionPool(host='namemc.com', port=443): Max retries exceeded with url: /profile/lol.7 (Caused by ConnectTimeoutError(<urllib3.connection.VerifiedHTTPSConnection object at 0x04D6DB38>, 'Connection to 181.143.106.162 timed out. (connect timeout=10)'))

**Comment by AXDZ** [comment]
yes i have been aware that this script is broken and because of my lazyness i haven't fixed it sorry :P

**Comment by qyis** [comment]
can you fix it please? when i type the name in and everything and hit enter it says ' Not supported proxy scheme None
Error has occured! ^'
